pkg/types/machinepools.go

```diff
@@ -160,8 +160,9 @@ type Fencing struct {
 
 // Credential stores the information about a baremetal host's management controller.
 type Credential struct {
-	HostName string `json:"hostName,omitempty" validate:"required,uniqueField"`
-	Username string `json:"username" validate:"required"`
-	Password string `json:"password" validate:"required"`
-	Address  string `json:"address" validate:"required,uniqueField"`
+	HostName                       string `json:"hostName,omitempty" validate:"required,uniqueField"`
+	Username                       string `json:"username" validate:"required"`
+	Password                       string `json:"password" validate:"required"`
+	Address                        string `json:"address" validate:"required,uniqueField"`
+	DisableCertificateVerification bool   `json:"disableCertificateVerification"`
```

Comment:
I generally try to follow API team's guiedlines of never using booleans, instead opting for enums like `Enabled` `Disabled`. So take that if it's useful, but I'm fine with this.

Reply:
Thanks, good feedback, I'll keep that in mind for future changes.
In this particular case I was just trying to stay aligned with the [BMC implementation](https://github.com/openshift/installer/blob/main/pkg/types/baremetal/platform.go#L15)